user/caching.md

```diff
@@ -147,7 +147,7 @@ cache:
   - $HOME/.m2
 ```
 
-As you can see, it is also possible to use environment variables in the directories.
```

Comment:
I know this isn't part of your change, but while we're here, can we change this to something like:

> You can use environment variables as part of the directory path

So it does not sound like something you can do inside the directory.